ui/framework/src/ui-framework/oidc/SignIn.tsx

```diff
@@ -44,16 +44,13 @@ export class SignIn extends React.PureComponent<SignInProps> {
   }
 
   public componentDidMount() {
-    const oidcClient = IModelApp.authorizationClient;
     // istanbul ignore if
-    if (isFrontendAuthorizationClient(oidcClient))
-      this._oidcClient = oidcClient;
+    if (IModelApp.authorizationClient !== undefined)
```

Comment:
- No need to test for (IModelApp.authorizationClient !== undefined) before setting this._oidcClient.
- change this._oidcClient. uses to  this._oidcClient?.

Reply:
Fixed. 

> > > Can we make ElectronAuthorizationBackend public too? We are using it for many command line utilities, as are third parties.
> > 
> > 
> > It's not just ElectronAuthorizationBackend, but everything around Electron/Desktops - I did check on it - we should just take that up separately.
> 
> I'm less concerned about the Electron use case than command line - this is widely enough used I'd really like to not see it change again. Can we just make this class public?

I think that's true for all of Electron/Desktop - we need to turn all that to be public - all I'm saying is lets just take that up as a separate PR. The ElectronAuthorizationBackend doesn't follow the pattern of other OIDC Clients anymore anyway.